.github/workflows/benchmarks-pr.yml

```diff
@@ -20,78 +20,78 @@ jobs:
     - uses: actions/checkout@v3
       with:
         ref: ${{ github.pull_request.head.sha }}
-    - uses: actions/setup-dotnet@v3
+    - uses: actions/setup-dotnet@v1
```

Comment:
I wonder the reason to roll back its version.

Reply:
just mistake 😓  I'll fix this mistake